src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int indexBall = new Random().nextInt(100);
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Similarly, 'ColorSupplier' should be instantiated once and reused, rather than creating a new instance each time a ball is generated.

Reply:
Fixed